src/services/TeamHub/TeamHub.BLL/Contracts/ICommentService.cs

```diff
@@ -0,0 +1,9 @@
+using TeamHub.DAL.Models;
+
+namespace TeamHub.BLL.Contracts
+{
+    public interface ICommentService
+    {
+        Task<Comment> GetCommentAsync(int commentId, CancellationToken cancellationToken);
+    }
```

Comment:
Rename the method in the service. This service is used to check for existence and return an object. You can rename it to: GetCommentAndCheckIfItExistsAsync.
Additionally, it's better to change the service name to make it explicitly clear what its purpose will be